yellowbrick/text/dispersion.py

```diff
@@ -84,13 +88,25 @@ def fit(self, text):
             A list of words in the order they appear in the corpus.
         """
             
+        # Concatenating documents and define boundaries by word offset
+        self.boundaries_ = []
+        self.doc_words_ = []
+        self.offset = 0
+
+        for doc in text:
+            for word in doc.split():
```

Comment:
I think it's reasonable to expect as input a list of lists; e.g. a list of documents where each document is a list of words as opposed to a list of strings. This is closer to what scikit-learn does in `sklearn.feature_extraction.text`, which we'll want to use eventually in this module, e.g. #488 and will also allow the user to use `nltk.word_tokenize` or `nltk.bigrams` if required. 

Therefore I recommend changing this to `for word in doc:` rather than simply splitting on space.